### ISSUE TITLE
bump commit doesn't work as expected, bumps twice

### ISSUE THREAD

**Issue by logicminds** [voxpupuli/puppet-blacksmith#22]
I am not sure why it bumps the version twice.  Am I don't this wrong or is the task broken?
```
bundle exec rake module:bump:minor
(in /Users/cosman/github/gitlab_mirrors)
Bumping version from 0.4.0 to 0.5.0
Bumping version from 0.5.0 to 0.6.0

```

**Comment by igalic** [comment]
this was fixed in 298fd142853c5a0f9a23aab5a2a0b6399f319e76 and released as https://github.com/maestrodev/puppet-blacksmith/releases/tag/v3.3.1

**Comment by logicminds** [comment]
Whoa, that was fixed a while ago.  Let me check to see which version of blacksmith I am using.
